src/main/java/org/apache/accumulo/testing/continuous/CreateTable.java

```diff
@@ -55,9 +57,20 @@ public static void main(String[] args) throws Exception {
         System.exit(-1);
       }
 
+      // retrieve and set tserver props
+      Map<String,String> props = getProps(env, TestProps.CI_COMMON_ACCUMULO_SERVER_PROPS);
+      try {
+        for (Map.Entry<String,String> entry : props.entrySet()) {
+          client.instanceOperations().setProperty(entry.getKey(), entry.getValue());
+        }
```

Comment:
Looking back at this, I'm wondering if an `instanceOperations().setProperties()` method would be useful. Haven't looked into how often it could be used but it would be clean up cases like this where multiple properties need to be set.